pkg/webhook/config.go

```diff
@@ -25,14 +25,31 @@ import (
 // Config defines the set over environment variables required
 // for running this application.
 type Config struct {
-	Port          string `env:"PORT,default=8080"`
-	ProjectID     string `env:"PROJECT_ID,required"`
-	TopicID       string `env:"TOPIC_ID,required"`
-	WebhookSecret string `env:"WEBHOOK_SECRET,required"`
+	BigQueryProjectID    string `env:"BIG_QUERY_PROJECT_ID,default=$PROJECT_ID"`
```

Comment:
We've reached the size where we probably need to document these, specifically with co-dependent behavior like BQProjectID defaulting to PROJECT_ID.

Reply:
ack adding to readme.